<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>music</class>
 <widget class="QWidget" name="music">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>450</height>
    </rect>
   </property>
   <widget class="QLabel" name="state">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>15</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="image/image.qrc">
      <normaloff>:/timg.ico</normaloff>:/timg.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>600</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit3_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>256</width>
      <height>241</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>歌曲列表</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.凡人歌</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2.成都</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3.左手指月</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4.差三岁</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5.盗将行</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton1_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>290</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>播放</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton3_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>330</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>上一首</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>260</y>
      <width>72</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>音量控制</string>
    </property>
   </widget>
   <widget class="QSlider" name="Slider_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton2_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>290</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton4_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>330</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>下一首</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monotype Corsiva</family>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>music</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image/image.qrc"/>
 </resources>
 <connections/>
</ui>
